scripts/create-kubestellar-demo-env.sh

```diff
@@ -17,6 +17,14 @@
 
 set -e
 
+
+echo -e "Checking container runtime..."
+if ! docker ps -q; then
```

Comment:
FYI: `-q` does not suppress all output.

Reply:
@MikeSpreitzer Should I do something like this instead? This was my original idea. To my knowledge null device changes between linux/macos and windows. 

```bash
# Need to check if windows / linux&macos. 
if [[ "$OSTYPE" == "msys" || "$OSTYPE" == "cygwin" || "$OSTYPE" == "win32" ]]; then
    # windows (Git bash, Cygwin, or native Windows)
    NULL_DEVICE="NUL"
else
    NULL_DEVICE="/dev/null"
fi

if ! docker ps -q > "$NULL_DEVICE" 2>&1; then
    echo "Error: The script cannot continue because Docker or Podman is not running. Please start your container runtime before running the script again."
    # You can add any additional actions here if Docker is not running
fi

```

Reply:
(BTW, if you are discarding the output then `-q` doesn't matter, right?)

I am not familiar with whether/what cygwin and msys have instead of /dev/null. Assuming that what you have written is accurate, that would work.

It _is_ a dismaying lot of code. Could we instead unify on directing output to a shell variable? Something like the following?

```shell
if ! dunsel=$(docker ps); then
    ...
fi
```

Reply:
You're right, this is a much simpler way to do it, and its easier to make sure it works across all devices.